shared/src/test/scala/pl/metastack/metatime/TimeTest.scala

```diff
@@ -18,13 +18,13 @@ object TimeTest extends SimpleTestSuite with TestIgnored {
     assertEquals(Hour(25).days.truncate, Day(1))
   }
 
-  testIgnored("operators") {
-    assertEquals(Time(23, 10) > Time(22, 0), true)
-    assertEquals(Time(23, 10) > Hour(22), true)
+  test("operators") {
+    assertEquals(Time(20, 10) < Time(22, 0), true)
+    assertEquals(Time(14, 10) < Hour(22), true)
   }
 
-  testIgnored("now") {
-    assertEquals(Time.now(), ???)
+  test("now") {
+    assertEquals(Time(0,0,0,0) < Time.now(), true)
```

Comment:
You could call `Time.now().unix()`, then wait a second and call it again. Afterwards check whether the difference is `<= 1.second.unix()`.